style.css

```
@@ -100,11 +104,61 @@ hr {
   opacity: 20%;
 }
 
+.imagens-redes:hover {
+  opacity: 30%;
+  transition: 0.3s;
+}
+
+.imagens-redes:active {
+  transition: 0.3s;
+  opacity: 40%;
+}
+
 .area-2 {
   background-color: #e7ebf8;
+}
+
+.tecnologias, .projetos,
+.experiencia,
+.formacao {
   padding: 1.5rem;
 }
 
+.tecnologias {
+  background-color: #f1f3fa;
+}
+
+.programacao {
+  margin: 1rem 0;
+}
+
+.programacao_grupo1, .programacao_grupo2 {
+  display: flex;
+  justify-content: space-around;
+}
+
+.programacao_grupo1 {
+  margin-bottom: 3rem;
+}
```

pq vc não usa um helper pra fazer essa margem e única essas seções programcao_grupo? <br>
Por exemplo 
```suggestion
.programacao_grupo {
  display: flex;
  justify-content: space-around;
}

.mb-3 {
  margin-bottom: 3rem;
}
```

Lembrando de mudar quem usa essas classes!

---

ps, vc usou `_` e no resto do css vc usou `-`, isso é confuso